example_main_test.go

```diff
@@ -0,0 +1,54 @@
+package promptui
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+)
+
+// This is an example for the Prompt mode of promptui. In this example, a prompt is created
+// with a validator function that validates the given value to make sure its a number.
+// If successful, it will output the chosen number in a formatted message.
+func Example() {
```

Comment:
Name this `Example_prompt` and then in the godoc it will display as `Example (Prompt)`